README.md

```diff
@@ -2,18 +2,21 @@
 Gazebo code for interfacing with the NUbots system
 
 # Setup
-  1. Install [Gazebo](http://gazebosim.org/tutorials?cat=guided_b&tut=guided_b1) (version 9 last tested).
-  2. Clone this repo: 
-  
+  1. Install [Gazebo](http://gazebosim.org/tutorials?cat=guided_b&tut=guided_b1) (tested with versions 9 and 10).
+  2. Clone this repo:
+
     $ git clone git@github.com:NUbots/Gazebo.git
-    
+
   3. Install Plugin:
-  
-    $ cd Gazebo 
-    $ bash install_plugin.sh
-   
+
+    $ cd Gazebo
+    $ mkdir build
+    $ cd build
+    $ cmake .. -GNinja
+    $ ninja install
```

Comment:
The other option for this is that we provide a way to run with gazebo plugin path environment variable set and a path to the models so we don't have to install the plugin into gazebo itself